apps/cow-amm-deployer/src/app/new/(components)/SushiPriceChecker.tsx

```diff
@@ -0,0 +1,71 @@
+import { toast } from "@bleu-fi/ui";
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk";
+import { UseFormReturn } from "react-hook-form";
+import { Address } from "viem";
+
+import { Input } from "#/components/Input";
+import { pairs } from "#/lib/gqlSushi";
+import { ammFormSchema } from "#/lib/schema";
+
+export function SushiV2PriceChecker({
+  form,
+}: {
+  form: UseFormReturn<typeof ammFormSchema._type>;
+}) {
+  const { register, setValue, watch } = form;
+  const {
+    safe: { chainId },
+  } = useSafeAppsSDK();
+
+  const token0 = watch("token0");
+  const token1 = watch("token1");
+  const tokenAddresses = [token0?.address, token1?.address].filter(
+    (address) => address,
+  ) as Address[];
+  return (
+    <div className="flex flex-col gap-y-1">
+      <Input
+        label="Sushi V2 Pool Address"
+        {...register("sushiV2Pair")}
+        tooltipText="The address of the Uniswap V2 pool that will be used as the price oracle. If you click on the load button it will try to find the most liquid pool address using the Uniswap V2 subgraph."
+      />
+      <button
+        type="button"
+        className="flex flex-row outline-none hover:text-highlight text-xs"
+        onClick={() => {
+          getSushiV2PairAddress(chainId, tokenAddresses[0], tokenAddresses[1])
+            .then((address) => {
+              setValue("sushiV2Pair", address);
+            })
+            .catch(() => {
+              toast({
+                title: "Pair not found",
+                description:
+                  "None Sushi V2 Pair with at least $1000 TVL was found for the selected tokens.",
```

Comment:
```suggestion
                  "No Sushi V2 Pair with at least $1,000 TVL was found for the selected tokens.",
```

should we standardize these texts and build them in a function?